core/src/main/java/pl/agh/edu/ui/screen/main/ScenarioScreen.java

```diff
@@ -0,0 +1,92 @@
+package pl.agh.edu.ui.screen.main;
+
+import java.util.Optional;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.InputMultiplexer;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+import pl.agh.edu.GdxGame;
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.engine.hotel.HotelType;
+import pl.agh.edu.engine.hotel.dificulty.DifficultyLevel;
+import pl.agh.edu.engine.hotel.dificulty.GameDifficultyManager;
+import pl.agh.edu.ui.GameSkin;
+import pl.agh.edu.ui.panel.DifficultyPanel;
+import pl.agh.edu.ui.panel.ScenarioPanel;
+
+public class ScenarioScreen implements Screen {
+	public final Stage stage = new Stage(GraphicConfig.getViewport());
+	public final GdxGame game = (GdxGame) Gdx.app.getApplicationListener();
+	public final Skin skin = GameSkin.getInstance();
+	public final ScenarioPanel scenarioPanel = new ScenarioPanel(this::goToDifficultyPanel);
```

Comment:
jeszcze widzę że to trochę zaburza wyglad tego screena i może warto przerzucić tą fukcjonalnać do innej klasy coś w stylu "GameCreationWizard", ale up to you